query/src/main/scala/filodb/query/exec/BinaryJoinExec.scala

```diff
@@ -50,7 +50,8 @@ final case class BinaryJoinExec(id: String,
   require(!on.contains(metricColumn), "On cannot contain metric name")
 
   val onLabels = on.map(Utf8Str(_)).toSet
-  val ignoringLabels = ignoring.map(Utf8Str(_)).toSet + metricColumn.utf8
+  val ignoringLabels = ignoring.map(Utf8Str(_)).toSet
+  val ignoringLabelsForJoin = ignoring.map(Utf8Str(_)).toSet + metricColumn.utf8
```

Comment:
Can the above be `ignoringLabelsForJoin = ignoringLabels + metricColumn.utf8` ?  Reads better and saves some computation

Reply:
Good suggestion! Made the change